src/parser/hunter/shared/modules/core/DeathTracker.tsx

```diff
@@ -0,0 +1,32 @@
+import CoreDeathTracker from 'parser/shared/modules/DeathTracker';
+
+
+/**
+ * Due to combatlog restrictions it is not possible to know whether the hunter cast Feign Death or the hunter actually died.
+ * For this module we assume any single death lasting less than 0.2% of an encounter (1.2 seconds on a 10 minute fight) was due to Feign Death, and thus we remove it as it is improbable anyone is that fast to ress anyone.
+ */
+const TIME_SPENT_DEAD_THRESHOLD = 0.002; //0.2%
+// @ts-ignore duplicate indicator, but it's extended just like other modules
+class DeathTracker extends CoreDeathTracker {
```

Comment:
I'm really confused about this ts-ignore, and I don't know why it's necessary - but it kept insisting there was a duplicate identifier..
![image](https://user-images.githubusercontent.com/29204244/83699250-d95bb480-a603-11ea-90f3-56bed953f2e4.png)